doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1786,6 +1786,31 @@ JSON files for `Creative Commons licenses <https://creativecommons.org/about/ccl
 
 .. _adding-custom-licenses:
 
+Adding Software Licenses
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+JSON files for software licenses are provided below.
+
+- :download:`licenseMIT.json <../../../../scripts/api/data/licenses/licenseMIT.json>`
+- :download:`licenseApache-2.0.json <../../../../scripts/api/data/licenses/licenseApache-2.0.json>`
+
+Contributing to the Collection of Standard Licenses Above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you do not find the license JSON you need above, you are encouraged to contribute it to this documentation. Following the Dataverse 6.2 release, we have standardized on the following procedure:
+
+- Look for the license at https://spdx.org/licenses/
+- ``cd scripts/api/data/licenses``
+- Copy an existing license as a starting point.
+- Name your file using the SPDX identifier. For example, if the identifier is ``Apache-2.0``, you should name your file ``licenseApache-2.0.json``.
+- For the ``name`` field, use the "short identifier" from the SPDX landing page (e.g. ``Apache-2.0``).
+- For the ``description`` field, use the "full name" from the SPDX landing page (e.g. ``Apache License 2.0``).
+- For the ``uri`` field, go to the SPDX landing page for the license and remove the extension ``.html`` from your browser and then copy the URL (e.g. ``https://spdx.org/licenses/Apache-2.0``) into the ``uri`` field.
```

Comment:
@jp-tosca at standup this morning @qqmyers said something about going to DataCite Fabrica and doing a search for the license using its SPDX identifier. And from there we could/should use the URL that DataCite uses (presumably from any of the returned results).

I'm not sure if I'm doing this right but if you type `rightsList.rightsIdentifierScheme:SPDX AND rightsList.rightsIdentifier:Apache-2.0` at https://commons.datacite.org you'll find a number of works that are licensed under Apache 2.

Maybe this is a starting point? Or the wrong direction? I'm not sure. For Apache 2, DataCite is giving back two different URLs when you export the metadata.

Reply:
I meant going to https://doi.test.datacite.org/ (with a valid account) and creating a DOI using their form. They show a list from spdx and then pick a specific URI for you (the rightUri field is greyed out/non-editable if you pick a license from the list): 
![image](https://github.com/IQSS/dataverse/assets/6731983/8fcb17ec-2fa6-45a8-b516-4eb5836c92de)
which is what eventually shows in their DataCite XML. They are not using the spdx URI itself for Apache2, CC or MIT at least.

Reply:
@qqmyers thank. I had a play with it. Check this out. They seem to simply return [the first](https://github.com/datacite/bracco/blob/670f34a0d872cb90492cbb641caee5bab3c6f122/app/components/doi-rights.js#L29C9-L29C59) in an array...

`this.fragment.set('rightsUri', rights.seeAlso[0]);`

So for the [EFL-2.0](https://github.com/datacite/bracco/blob/670f34a0d872cb90492cbb641caee5bab3c6f122/app/spdx.js#L1385C1-L1398C7) license...

    {
      reference: './EFL-2.0.html',
      isDeprecatedLicenseId: false,
      isFsfLibre: true,
      detailsUrl: 'http://spdx.org/licenses/EFL-2.0.json',
      referenceNumber: '7',
      name: 'Eiffel Forum License v2.0',
      licenseId: 'EFL-2.0',
      seeAlso: [
        'http://www.eiffel-nice.org/license/eiffel-forum-license-2.html',
        'https://opensource.org/licenses/EFL-2.0'
      ],
      isOsiApproved: true
    },

... they don't show the opensource.org URL (like they do for MIT):

![Screenshot 2024-04-23 at 2 14 56 PM](https://github.com/IQSS/dataverse/assets/21006/d754b346-3307-4c08-b281-451442c770fb)

I guess I'm left wondering... is this array curated by DataCite to put the best URL first? Or it is fairly arbitrary?

Reply:
I think this one is an interesting example @pdurbin while searching on google to see what would be the canonical URL, searching for "Eiffel forum license 2.0" shows the first result as https://www.gnu.org/licenses/eiffel-forum-license-2.html, search for "eiffel license" then the first result is "https://www.eiffel.com/eiffelstudio/licensing/" so this URL "http://www.eiffel-nice.org/license/eiffel-forum-license-2.html" is not between the first results.

Reply:
Also as I mentioned on the call the second link that comes out of the JSON from Datacite is expired "https://opensource.org/licenses/EFL-2.0" shows an  **Oops! That page can’t be found.**

Reply:
Looks like SPDX isn't being kept up to date (or picked the wrong URL to start) - opensource now has that license at https://opensource.org/license/ver2_eiffel-php.